goalgorilla/open_social
# Issue #2875334 by maikelkoopman

Display image field default for new content types

Solution:
When a SM/developer creates a new content type, we do not know what the field name of the image will be, if they decide to name it image and this defaults to the machine name field_image or they reuse the image field of an existing type this solution will work and the image gets displayed. If they create a unique name, we cannot provide a template solution and manual theming is required.

We intend to create a follow up to look if we can use Display Suite module to allow a site builder role to use a new content type without having to add theming.

##how to test
- create a new content type
- add a image field and just call it image or reuse one of the existing image fields
- Configure the form and display
- Create new content
- Add the CT to an existing view (for example all-topics)
- Check the view: the fields should display like the other existing content type (image on the left)

https://www.drupal.org/node/2875334
